### PR TITLE
Add solution and unit test for problems.

### PR DESCRIPTION
**Pull request by BrianNguyen420**

### CODE REVIEW

File: simpleSequential/test_booleanToString.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import unittest
+from booleanToString import booleanToString
+
+class TestBooleanToString(unittest.TestCase):
+    def test_trueBooleanInput(self):
+        self.assertEqual(booleanToString(True), "True")
+
+    def test_FalseBooleanInput(self):
```
Comment (rubikvn2100) [inline simpleSequential/test_booleanToString.py]: Please fix this typo.

**Comment by rubikvn2100** [comment]
Please fix the commit message of commit “dd54b19”, the file name is wrong. Should be “test_booleanToString.py” not “booleanToString.py”.

File: simpleSequential/isPositiveAndDivisibleBy69.py [hunk 2]
```diff
@@ -11,3 +11,5 @@
 # Ex:
 #   isPositiveAndDivisibleBy69(690) # return True
 #   isPositiveAndDivisibleBy69(-69) # return False
+def isPositiveAndDivisibleBy69(var: int) -> bool:
+    return var > 0 and var % 69 == 0
```
Comment (rubikvn2100) [inline simpleSequential/isPositiveAndDivisibleBy69.py]: Great and correct solution. But one thing need to be changed to make thing better. Let create “isPositive” and “isDivisibleBy69” variable. Calculate the value for the variables, then use the variables in the expression when return. You can write the sudo code in comments here before start making change.

File: simpleSequential/test_isPositiveAndDivisibleBy69.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import unittest
+from isPositiveAndDivisibleBy69 import isPositiveAndDivisibleBy69
+
+class TestsIsPositiveAndDivisibleBy69(unittest.TestCase):
+    def test_positiveAndDivisibleBy69Input(self):
+        self.assertEqual(isPositiveAndDivisibleBy69(69), True)
+
```
Comment (rubikvn2100) [inline simpleSequential/test_isPositiveAndDivisibleBy69.py]: How about the case “positive and not divisible by 69”?

**Comment by rubikvn2100** [comment]
I think you need to reset all the 4 commits, fix, and recommit. Use:

1/ “git reset —soft HEAD~4” to soft reset the 4 commits.
2/ “git restore —staged <file name” and remove the 4 file (Note: you need to do it 4 times)
3/ fix the files.
4/ re commit them.
5/ “git push —force” to overwrite the pull request.

File: simpleSequential/isPositiveAndDivisibleBy69.py (hunk 2 above)
Comment (rubikvn2100) [reply simpleSequential/isPositiveAndDivisibleBy69.py]: Sorry, I think your is good enough, ignore my comments.

File: simpleSequential/test_booleanToString.py (hunk 1 above)
Comment (BrianNguyen420) [reply simpleSequential/test_booleanToString.py]: acknowledge.

**Pull request closed without merging**